tests/assoc_sim.R

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/Rscript
+#
+# Copyright 2015 Google Inc. All rights reserved.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+library("optparse")
+
+# First parse args
+options(stringsAsFactors = FALSE)
```

Comment:
The comments here are a little misleading.  options(stringsAsFactors) sets a global R option and doesn't have anything to do with flag parsing.  I would write it like this

options(...)

if(!interactive()) {
}

The comments on line 19 and 23 don't really add much and can be deleted.